### PR TITLE
Redux react components

### PR DESCRIPTION
**Pull request by juanmanuelbyc**
Used Redux store to display books. Also added some UI elements to components and made them dispatch actions.

### CODE REVIEW

File: src/components/categories.js [hunk 1]
```diff
@@ -1,10 +1,19 @@
...
+  };
+
   return (
     <div className="categoriesPageContainer">
       <h2>Categories</h2>
-      <button type="button" className="checkStatusButton">Check Status</button>
+      <button type="button" className="checkStatusButton" onClick={checkCats}>Check Status</button>
     </div>
   );
 }
```
Comment (fobadara) [inline src/components/categories.js]: [REQUIRED] The `under construction` text is not displayed. Kindly fix that.

| CATEGORIES PAGE |
|--------------------|
|bookstore1|

**Review by fobadara — CHANGES_REQUESTED** [review]
# REQUIRES CHANGES ♻️ 
Hi  @juanmanuelbyc, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

You can also consider:
- [OPTIONAL] Good job writing your commit messages in the imperative but in future make sure the first letter is in uppercase just the beginning of any normal sentence

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/categories.js (hunk 1 above)
Comment (Whoistolu) [reply src/components/categories.js]: Well resolved 👍🏽

**Review by Whoistolu — APPROVED** [review]
### Hi @juanmanuelbyc 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**